VPM.py

```diff
@@ -129,12 +132,10 @@ def compute_hop_length(win_length, overlap):
     Returns:
         hop_length (int): The computed hop_length.
     """
-    """
-    !! Write code here !!
-    """
+    return win_length*overlap
```

Comment:
```suggestion
    return int(win_length * (1 - overlap))
```
For example, with a 75% overlap for a win_length of 4, you should have a hop_length of 1